src/Xml/Decode.elm

```diff
@@ -92,19 +101,59 @@ parseSlice first firstClose trimmed =
 
         closeTag =
             "</" ++ tagName ++ ">"
+
+        openTag =
+            "<" ++ Regex.escape tagName ++ "[\\s\\n]*[^/]*?>" 
```

Comment:
Defining regexes at runtime like this can lead to runtime errors, they should instead be defined as flatly as possible, see `propRegex`. In the cases where they can't, I'd rather not use a regex for that.